src/test/resources/eopa.Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine:latest
+
+RUN apk add nginx bash
+
+ADD nginx.conf /etc/nginx/nginx.conf
+
+ADD entrypoint.sh /entrypoint.sh
+
+RUN chmod +x /entrypoint.sh
+
+# We name the eopa binary opa so that the entrypoint script does not need to be
+# modified.
+COPY --from=ghcr.io/styrainc/enterprise-opa:latest /ko-app/enterprise-opa-private /usr/bin/opa
```

Comment:
ℹ️ with the next release, this should no longer be necessary. In the mean time, you could use `:edge` here

Reply:
The binary still needs to be copied into the container one way or the other. We can't directly use the eopa container, because we need nginx inside it as well.